src/axolotl/kernels/rms_norm_gated.py

```
@@ -267,9 +280,45 @@ def rms_norm_gated_backward(dY, X, G, W, RSTD, offset, BLOCK_SIZE, num_warps):
+
+@_rms_norm_gated_bwd_op.register_fake
+def _(dY, X, G, W, RSTD, offset, BLOCK_SIZE, num_warps):
+    return torch.empty_like(dY), torch.empty_like(dY), torch.empty_like(W)
+
+
+def rms_norm_gated_forward(X, G, W, eps, offset):
+    shape = X.shape
+    dim = shape[-1]
+    X = X.view(-1, dim)
+    G = G.view(-1, dim)
+    n_rows, n_cols = X.shape
```

Ruff (0.15.20)

[warning] Unpacked variable `n_rows` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)